store/sqlstore/channel_store.go

```diff
@@ -1605,46 +1599,13 @@ func (s SqlChannelStore) GetMemberCount(channelId string, allowFromCache bool) (
 }
 
 func (s SqlChannelStore) InvalidatePinnedPostCount(channelId string) {
-	channelPinnedPostCountsCache.Remove(channelId)
-	if s.metrics != nil {
-		s.metrics.IncrementMemCacheInvalidationCounter("Channel Pinned Post Counts - Remove by ChannelId")
-	}
 }
 
 func (s SqlChannelStore) GetPinnedPostCountFromCache(channelId string) int64 {
```

Comment:
I think we should remove this method entirely, is not used anywhere and is not needed because we already have the `allowFromCache` param in the `GetPinnedPostCount`.